<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NonModualDialogClass</class>
 <widget class="QDialog" name="NonModualDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模式与非模式对话框</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnShow1">
       <property name="text">
        <string>模式对话框</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnShow2">
       <property name="text">
        <string>非模式对话框</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="nonmodualdialog.qrc"/>
 </resources>
 <connections/>
</ui>
